mgsr/app.js

```diff
@@ -0,0 +1,398 @@
+const gameState = {
```

Comment:
It's unfortunate the edit history is lost by copying. To get a better sense of which individual changes affected which things, I'd recommend actually looking at the individual commits (of which I made about 10), as they show where in your existing files I actually changed/added stuff.